docs/tldr/pages/linux/tmt.md

```diff
@@ -0,0 +1,37 @@
+# tmt
+
+> Test Management Tool for creating, running and debugging tests.
+> See also: `tmt-run`, `tmt-try`.
+> More information: <https://tmt.readthedocs.io>.
+
+- Look around to see what's available:
+
+`tmt`
+
+- Initialize tmt files/project structure:
+
+`tmt init`
+
+- Create a new test with template and link:
+
+`tmt test create --template beakerlib --link verifies:{{issue#1234}}`
+
+- List available tests, plans, or stories:
+
+`tmt <test|plan|story> ls {{pattern}}`
+
+- Show detailed test metadata in the given context:
+
+`tmt --context arch=aarch64 test show`
+
+- Validate tmt files against the specification:
+
+`tmt lint`
+
+- Use filter:
+
+`tmt tests ls --filter tag:{{foo}} --filter tier:{{0}}`
```

Comment:
```suggestion
`tmt tests ls --filter {{tag:foo}} --filter {{tier:0}}`
```

Would it make sense to place the whole filter values in the parenthesis? I would say it more clearly separates the values from the option.

Reply:
I guess could be either way, but would require changes in other entries, like the above arch=aarch64. I tried to use parentheses for things that are not defined keyword. Tag and Tier are core attributes, so I'm not sure, but no strong opinion.